spec/factories/links.rb

```diff
@@ -5,6 +5,10 @@
     url { "http://#{local_authority.slug}.example.com/#{service_interaction.service.slug}/#{service_interaction.interaction.label.parameterize}" }
     status nil
     link_last_checked nil
+
+    trait :with_service_interaction do
```

Comment:
This trait is probably not needed because we've already defined the service_interaction association for this factory. We tried removing this trait and any calls to it and the tests still passed.